onebusaway-android/src/main/java/org/onebusaway/android/nav/NavigationServiceProvider.java

```diff
@@ -83,6 +83,8 @@
     public boolean mFinished = false;   // Trip has finished.  //Change to public
     public boolean mGetReady = false;   // Get Ready triggered. //Change to public
 
+    public float sectoCurDistance = -1;
```

Comment:
Could you please change the name to mSectoCurDistance?

Reply:
Thank you for the reviewing! I have modified the variable name.